config/nrfconnect/overlay-sample-defaults.conf

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_CPLUSPLUS=y
+CONFIG_LIB_CPLUSPLUS=y
+CONFIG_NEWLIB_LIBC=y
+
+# Export POSIX names for functions implementing a subset of POSIX standard in Zephyr
+CONFIG_POSIX_API=y
+CONFIG_PTHREAD_IPC=y
+CONFIG_EVENTFD=y
+
+CONFIG_CHIP=y
```

Comment:
Maybe put that at the beginning since it's the most important setting?

Reply:
Done